### PR TITLE
Feature/login signup redirect

### PR DESCRIPTION
**Pull request by xokeixo**
## Description

As an admin, I want users to use Auth0 sign up.

#### Video Link

[Loom Video](https://www.loom.com/share/4b3d18273e0342a6a48088f24a19c34e)

#### Trello Link

https://trello.com/c/2AKQsh29

## Type of change

___Please delete options that are not relevant.___

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by williamvelichko — APPROVED** [review]
Moving towards auth0 authentication on step at a time so this is definitely needed. 
We need this merged because then we can keep moving forward with auth0 implementation.

File: src/components/pages/Navbar/NavbarFeatures/SignupButton.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function SignupButton() {
+  const { loginWithRedirect } = useAuth0();
+
   return (
-    <NavBtnLink to={`/signup`} key="signup">
+    <NavBtnLink
+      className="btn btn-primary btn-block"
+      onClick={() =>
```
Comment (kimnivore) [inline src/components/pages/Navbar/NavbarFeatures/SignupButton.js:11]: Nice job importing auth0 and getting the functionality to redirect and work correctly.

**Review by kimnivore — APPROVED** [review]

**Review by VABIII — APPROVED** [review]
That was a good explanation in your video about what you was done and why. Nice work.

**Review by aaronamendez — APPROVED** [review]
Looks good, great job here.

**Review by VABIII — APPROVED** [review]

**Pull request merged**